test/common/src/main/kotlin/org/codice/compliance/verification/core/ResponseProtocolVerifier.kt

```diff
@@ -84,11 +83,11 @@ class ResponseProtocolVerifier(val response: Node, val id: String) {
         verifyTimeValues(response.attributes.getNamedItem("IssueInstant"), SAMLCore_3_2_2_d)
 
         val destination = response.attributes?.getNamedItem("Destination")?.textContent
-        if (destination != ACS_URL)
+        if (destination != acsUrl)
             throw SAMLComplianceException.createWithPropertyNotEqualMessage(SAMLCore_3_2_2_e,
                     "Destination",
                     destination,
-                    ACS_URL)
+                    acsUrl ?: "NO_ACS")
```

Comment:
✏️ When I made my quick patch this morning, I just shoved this in so we'd have something to display. We might want to make it a constant or use some other string.